src/Hydration/Middleware/AddAttributesToRootTagOfHtml.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Livewire\HydrationMiddleware;
+namespace Livewire\Hydration\Middleware;
```

Comment:
Can you move all these back to their original directory?

Reply:
I think you are seeing an old version. It has already been moved.

Reply:
Github was showing old code. Has refreshed now so this comment is irrelevant.